gradle-recommended-product-dependencies/src/test/groovy/com/palantir/gradle/dist/RecommendedProductDependenciesPluginIntegrationSpec.groovy

```diff
@@ -95,14 +95,60 @@ class RecommendedProductDependenciesPluginIntegrationSpec extends IntegrationSpe
         runTasksSuccessfully('-DignoreDeprecations=true', '--write-locks', ':jar')
 
         then:
-        fileExists("build/libs/root-project.jar")
+        verifyCorrectRecommendedProductDeps('build/libs/root-project.jar')
+    }
+
+    def 'when gradle-shadow-jar is applied, the pdeps are put into the shadow jar manifest'() {
```

Comment:
is it worth adding a multi-project test here too to more closely mirror the a-c-j use-case?